stories/components/card/Card.mdx

```diff
@@ -41,10 +41,20 @@ For example, for 4 card columns on large screens and 3 columns on medium screens
 }
 ```
 
-By default, cards have a body and a footer.
+By default, cards are designed to be a list of links with a body and a footer.
 
 <Story of={stories.basic} />
 
 However, the card footer is optional.
 
 <Story of={stories.noFooter} />
+
+## Using cards as stylized content containers
+A card can be used flexibly as a content container that is not
```

Comment:
Perhaps I am parsing this too finely, but it seems to me that if the default behavior is that a card is a link then maybe we should be _adding_ a class to make it not a link?

Reply:
Good point. Updating this to add a `.card--container` class instead.